standalone-e2e-test/src/test/java/AbstractRestAssuredBase.java

```diff
@@ -0,0 +1,32 @@
+import configuration.RestAssuredTestConfiguration;
```

Comment:
I just realized that you haven't used some packages for your classes. Please use something like com.github.lammaer as your group id of your project.